kotlin-core/src/test/kotlin/com/segment/analytics/kotlin/core/utils/Plugins.kt

```diff
@@ -0,0 +1,54 @@
+package com.segment.analytics.kotlin.core.utils
+
+import com.segment.analytics.kotlin.core.*
+import com.segment.analytics.kotlin.core.platform.*
+
+class TestRunPlugin(override val name: String = "TestRunPlugin", var closure: (BaseEvent?) -> Unit): EventPlugin {
```

Comment:
Is this for tests?

Reply:
yup, added some docs to outline use-cases